ert_shared/ensemble_evaluator/ensemble/prefect.py

```diff
@@ -204,13 +204,14 @@ def evaluate(self, config: EvaluatorServerConfig, ee_id: str):
 
         # everything in self will be pickled since we bind a member function in target
         ctx = self._get_multiprocessing_context()
-        self._eval_proc = ctx.Process(target=self._evaluate)
-        self._eval_proc.daemon = True
-        self._eval_proc.start()
+        eval_proc = ctx.Process(target=self._evaluate)
+        eval_proc.daemon = True
+        eval_proc.start()
+        self._eval_proc = eval_proc
         self._allow_cancel.set()
 
     def _evaluate(self):
-        asyncio.set_event_loop(asyncio.get_event_loop())
+        get_event_loop()
```

Comment:
Does this do anything?

Reply:
I think it just ensures there is a loop on the current thread. Sometimes that is needed. Exactly why, I am unsure of. But I figured I would get all the code to use my new fancy get_event_loop() function.